app/components/global/ErrorPage.js

```diff
@@ -19,7 +19,7 @@ const Container = styled.div`
 `
 
 const Img = styled.img`
-  height: calc(${th('gridUnit')} * 4);
+  height: ${th('space.6')};
```

Comment:
This is the only instance where the value has changed ... i.e. old equates to 96px and new to 36px ... is this correct?

Reply:
Yes, old value is `96px` but new value is `72px`

Reply:
Surely 72px would need `space.12` (i.e. 6 * 12 = 72 )

Reply:
should nick be specifying 'x' instead of px?